README.md

```diff
@@ -19,7 +19,7 @@ This GitHub template is independent of the (research) data and its format. It fo
 
 ### Open research data
 
-- Citeable via [CITATION.cff](CITATION.cff) and [DOI](https://www.doi.org/)
+- Citeable via [DOI](https://www.doi.org/) (and [CITATION.cff](CITATION.cff))
```

Comment:
```suggestion
- Citeable via [DOI](https://www.doi.org/) (and [CITATION.cff](https://citation-file-format.github.io/))
```
(Replaced link to `CITATION.cff` since the file is deleted)